### PR TITLE
110/Better support for localization

### PR DESCRIPTION
**Pull request by friedger**
This PR just changes some strings.

* Replace strings with resource ids
* Use locale in String.format

### CODE REVIEW

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/133/
Test PASSed.

File: library/src/main/java/com/novoda/downloadmanager/lib/Query.java [hunk 1]
```diff
@@ -31,7 +32,7 @@
      */
     public static final int ORDER_DESCENDING = 2;
 
-    private static final String ORDER_BY_LIVENESS = String.format(
+    private static final String ORDER_BY_LIVENESS = String.format(Locale.US,
```
Comment (amlcurran) [inline library/src/main/java/com/novoda/downloadmanager/lib/Query.java]: Do we still want to have a locale, given this is a SQL string?

Comment (friedger) [inline library/src/main/java/com/novoda/downloadmanager/lib/Query.java]: It is recommended to use US, instead of Locale.getDefault()

Comment (amlcurran) [inline library/src/main/java/com/novoda/downloadmanager/lib/Query.java]: oh of course, I was getting confused

**Comment by amlcurran** [comment]
LGTM, any more :eyes:?

File: library/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string name="dl__cancel">Cancel</string>
+  <string name="dl__download_cancelled">Download cancelled.</string>
+  <string name="dl__download_unsucessful">Download unsuccessful.</string>
```
Comment (ronocod) [inline library/src/main/res/values/strings.xml]: Typo here, should be `dl__download_unsuccessful`

Comment (friedger) [inline library/src/main/res/values/strings.xml]: nice spot. fixed.

**Comment by friedger** [comment]
fixes #110

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/134/
Test PASSed.

**Comment by blundell** [comment]
:+1: @friedger https://github.com/novoda/download-manager/blob/a264c5e8d040121e89b0e351f6eb884bb565370b/library/src/main/java/com/novoda/downloadmanager/lib/NotificationDisplayer.java#L154 fyi

**Comment by friedger** [comment]
@blundell I thought this does not need to be localised but you are right.

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/135/
Test FAILed.

**Comment by hal9002** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://ci.novoda.com/job/download-manager-prb/136/
Test PASSed.

**Comment by friedger** [comment]
Looks like sometimes this is used in arabic: http://unicode-table.com/en/066A/

**Comment by blundell** [comment]
:+1:

**Pull request merged**